### DISCUSSION TITLE
Can `nixd` lsp be configured so that it can show suggestion for my standalone nixvim (`makeNixvimWithModule`) ?

### DISCUSSION THREAD

**Discussion by amalgame21** [nix-community/nixvim#2571] [Q&A]
I just watch the video on youtube from vimjoyer (id=`M_zMoHlbZBY`, title=`Easiest Way To Write Nix | Code Editor Setup`)
It would be very helpful to edit nixvim config if it is possible.

**Comment by MattSturgeon** [comment]
I'll try and remember to come back with more detail, but in short yes.

I do this here in my config: https://github.com/MattSturgeon/nix-config/blob/main/nvim%2Fconfig%2Flsp.nix#L38
